File: backend/rs-client/php/src/service/SiteCreativeApprovalService.php
<?php

/**
 
 */
class SiteCreativeApprovalService {

    private $configuration;

    public function __construct(RsConfiguration $configuration) {
        $this->configuration = $configuration;
    }

    
    public function get (array $selector) {
        $client = $this->configuration->createClient();
        $parameters = null;
        $data = null;
    
        $parameters = $selector;
        
    
        return $client
                ->request("GET" ,"/siteCreativeApprovals/", $parameters, $data);
    }
    
    public function perform ($operations) {
        $client = $this->configuration->createClient();
        $parameters = null;
        $data = null;
    
        
        $data = $operations;
    
        return $client
                ->request("POST" ,"/siteCreativeApprovals/", $parameters, $data);
    }
}

File: backend/rs-client/java/src/main/java/com/foros/rs/client/service/SiteCreativeApprovalService.java
package com.foros.rs.client.service;

import com.foros.rs.client.RsClient;
import com.foros.rs.client.data.JAXBEntity;
import com.foros.rs.client.model.siteCreative.SiteCreativeApproval;
import com.foros.rs.client.model.siteCreative.SiteCreativeApprovalOperations;
import com.foros.rs.client.model.siteCreative.SiteCreativeApprovalSelector;

public class SiteCreativeApprovalService extends ReadonlyServiceSupport<SiteCreativeApprovalSelector, SiteCreativeApproval> {

    public SiteCreativeApprovalService(RsClient rsClient) {
        super(rsClient, "/siteCreativeApprovals");
    }

    public void perform(SiteCreativeApprovalOperations operations) {
        rsClient.post(path, new JAXBEntity(operations));
    }
}